src/bindings/config.ts

```diff
@@ -98,6 +99,10 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     wallet: {
       registerWalletWithVerification: registerWalletWithVerification,
     },
+    ask: {
+      apiKey: process.env.OPENAI_API_KEY ?? "",
+      tokenLimit: Number(process.env.OPENAI_TOKEN_LIMIT) || DEFAULT_TOKEN_LIMIT,
```

Comment:
This token limit is not used anywhere. Should we delete it?

Reply:
I put it in so that there is some control over it but didn't want to impose a limit myself. We are limited to ~8k tokens when using gpt-4 unless going with the ~32k api, in my tests when pulling context from current and external sources (in my own very small issues/prs) I was hitting 4k tokens easily with both input and output combined, so I fully expect it to be a requirement to manage the input tokens at least for much bigger context

Reply:
no I mean that with both input and output combined we have a max of ~8k tokens

Reply:
The gpt response object currently returns the token usage information so it can be gauged during these stages for what might be an acceptable ratio for input vs output allocation

Reply:
Where should the `tokenLimit` param be used? I mean where we should pass it exactly in the code.
```js
  const openAI = new OpenAI({
    apiKey: config.ask.apiKey,
  });
```

Here during openai init?

Reply:
@Keyrxng pls move the "token limit" param to the bot's config and set it to some reasonable value